src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -83,9 +86,28 @@ describe('Facility search', () => {
 
   it('does not show search result header if no results are found', () => {
     cy.visit('/find-locations?fail=true');
+
+    cy.get('#facility-search-results').should('not.exist');
+  });
+
+  it('finds community dentists', () => {
+    cy.visit('/find-locations');
+
```

Comment:
You should mock the correct data and verify

Do something like this:

```    cy.route(
      'GET',
      '/v1/facilities/cpp/',
      'fx:constants/<CCP_MOCK_DATA>',
    ).as('searchCCP');``

Reply:
I was able to use the `MockApi` instead by checking for `window.Cypress` in `src/applications/facility-locator/api/index.js`.